sources/packages/api/src-sql/DisbursementSchedules/Alter-document-number-col.sql

```diff
@@ -0,0 +1,4 @@
+ALTER TABLE
```

Comment:
Can we please add some comments at the top ?
--Drop the NOT NULL constraint on document_number. Similarly to the rollback file too.